siriuspy/siriuspy/devices/fofb.py

```diff
@@ -1575,6 +1576,15 @@ def fofbacc_filter_gain(self):
     def fofbacc_filter_gain(self, value):
         self['FOFBAccFilterGain-SP'] = value
 
+    @property
+    def psconfig_matrix(self):
```

Comment:
maybe we could use the same abbreviation used in `respmat`, so this would be `psconfig_mat` ?